src/latexify/frontend.py

```diff
@@ -10,58 +10,41 @@
 from latexify import exceptions
 from latexify import parser
 from latexify import transformers
+from latexify.config import Config
```

Comment:
Google's style guide does not allow direct imports.

https://google.github.io/styleguide/pyguide.html#22-imports

```suggestion
from latexify import config
```

Reply:
sorry, I don't know how to modify this one.
Could you help me?

Reply:
you'll have to change the import, and change any uses of `Config` to `config.Config`

Reply:
```python
def get_latex(
    fn: Callable[..., Any], *, _config: config.Config | None = None, **kwargs
) -> str:
```

Thanks for your reply. Is it looks like this one?

Reply:
Ah yeah that makes it a bit awkward for the keyword argument doesn't it.

Maybe change the import to `from latexify import config as cfg`?

That way we can keep the name of the keyword argument as `..., config: cfg.Config | None = None, ...`.

Reply:
Thanks for your advice, you were right